backend/spec/requests/books_spec.rb

```diff
@@ -6,9 +6,12 @@
 RSpec.describe("Books") do
   let(:data) { json_body[:data] }
   let(:error_messages) { json_body[:errors] }
-  let(:headers) { generate_authorization_header }
+  let(:headers) { generate_authorization_header(User.first) }
```

Comment:
nit, just get the user that you want by finding them via name or something. Just an advice .first is hooked to the primary key, which when you have more complicated tests would probably shock you that the bug was in .first (happened recently to us lol)